flaky-testing.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/zsh
+
+for i in `seq 1 1000`; do
+  pytest -v -s tests/power_distribution/test_power_distributor.py >/dev/null 2>&1
+  echo "$?"
+done
```

Comment:
I guess this is a temporary script that you want to keep out of git?

Reply:
Yep. Removed.